test/inlining/applyBugs.baseline

```diff
@@ -0,0 +1,12 @@
+PASSED
+
+PASSED
+
+PASSED
+
+PASSED
+
+PASSED
+1
+1
+1

```

Comment:
nit: can you check this value array inside the script itself? - and avoid this baseline file ?